mmv1/products/firebase/api.yaml

```diff
@@ -104,7 +104,7 @@ objects:
     name: 'WebApp'
     min_version: beta
     base_url: projects/{{project}}/webApps
-    self_link: '{{name}}'
+    self_link: '{{project}} {{name}}'
```

Comment:
To mirror monitoring log config you'll want to keep this as `name` and make the change to `import_format` directly, I think.

Can the `appId` include slashes? It's not clear why we need to do this instead of improving the `id`.

Reply:
I don't track. We need to change the `self_link` to include project. That way when import runs it will pick up project. 

https://github.com/GoogleCloudPlatform/magic-modules/blob/master/mmv1/templates/terraform/custom_import/self_link_as_name.erb#L5

We don't need to change the import format since the `self_link_as_name` already looks for `{{project}} {{name}}`

Reply:
Ah I think I see

Reply:
Changing the import format'll change the documentation, this theoretically already works today

Reply:
Note that to fix the test we'll need to send the custom import id, not sure if examples can do that (which is why if `appId` can't contain slashes, actually fixing the root cause in the id/import format is preferable)

Reply:
I've changed the import format for the documentation. This fixes the test because we don't need custom import id since the id/self_link is equal to the value above. 

I tried for about 2 hours to get the app_id stuff working and ran into a ton of various road blocks. I don't think it's worth spending too much time on it to further unblock that pathways vs the value added.

Reply:
Can you post the results of a passing test? I am fairly confident that this will create a malformed URL, and the test will fail.

Reply:
Yep, you are right. 🤦🏻 

Okay. After reading further into it we should be safe to just ignore the project here.
https://github.com/hashicorp/terraform-provider-google-beta/blob/master/google-beta/resource_firebase_web_app.go#L156-L159

It's not actually being used in the official FETCH of the resource or update call. So the import bug is purely visual since the project isn't used anywhere outside of create.